CONTRIBUTING.md

```diff
@@ -51,47 +51,21 @@ npm install -g bower
 
 # Install Bower dependencies
 bower install
-
-# Create a .env file and populate it with the necessary API keys and secrets:
-touch .env
 ```
-
-Edit your `.env` file with the following API keys accordingly. If you only use email login, only the `MONGOHQ_URL`, `SESSION_SECRET`, `MANDRILL_USER` and `MANDRILL_PASSWORD` fields are necessary. Keep in mind if you want to use more services you'll have to get your own API keys for those services. If you only use a subset or no OAuth2 authentication methods, you may want to remove them from ```server/passport-providers.js``` - otherwise the server will complain about missing clientIDs at launch.
-
+*Private Environment Variables (API Keys)*
+```bash
+# Create a copy of the "sample.env" and name it as ".env".
+# Populate it with the necessary API keys and secrets:
+cp sample.env .env
 ```
-MONGOHQ_URL='mongodb://localhost:27017/freecodecamp'
-
-FACEBOOK_ID=stuff
-FACEBOOK_SECRET=stuff
-
-GITHUB_ID=stuff
-GITHUB_SECRET=stuff
 
-GOOGLE_ID=stuff
-GOOGLE_SECRET=stuff
+Edit your `.env` file and modify the API keys only for services that you will use.
 
-LINKEDIN_ID=stuff
-LINKEDIN_SECRET=stuff
+Note : Not all keys are required, to run the app locally, however `MONGOHQ_URL` is the most important one.
 
-MANDRILL_PASSWORD=stuff
-MANDRILL_USER=stuff
+If you only use email login, only the `MONGOHQ_URL`, `SESSION_SECRET`, `MANDRILL_USER` and `MANDRILL_PASSWORD` fields are necessary.
```

Comment:
I would clarify that point because I don't have this set and it works fine for most purposes.  I'm not even sure where I would get a MANDRILL_USER/PASSWORD.  If we know where to get that, we should maybe include it in the docs.